openstates/id/bills.py

```diff
@@ -1,94 +1,93 @@
+                         'Res/Env': 'Resources and Environment',
+                         'St Aff': 'State Affairs', 'Transp': 'Transportation'}}
 
 # a full list of the abbreviations and definitions can be found at:
 # http://legislature.idaho.gov/sessioninfo/glossary.htm
 # background on bill to law can be found at:
 # http://legislature.idaho.gov/about/jointrules.htm
 _ACTIONS = (
-     # bill:reading:1
-     (r'(\w+) intro - (\d)\w+ rdg - to (\w+/?\s?\w+\s?\w+)',
-      lambda mch, ch: ["bill:introduced", "bill:reading:1", "committee:referred"] \
-                        if mch.groups()[2] in _COMMITTEES[ch] else ["bill:introduced", "bill:reading:1"] ),
-     # committee actions
-     (r'rpt prt - to\s(\w+/?\s?\w+)',
-      lambda mch, ch: ["committee:referred"] if mch.groups()[0] in _COMMITTEES[ch] \
-                                             else "other"),
-     # it is difficult to figure out which committee passed/reported out a bill
-     # but i guess we at least know that only committees report out
-     (r'rpt out - rec d/p', "committee:passed:favorable"),
-     (r'^rpt out', 'committee:passed'),
-
-
-    (r'^Reported Signed by Governor', "governor:signed"),
-    (r'^Signed by Governor', "governor:signed"),
-
-     # I dont recall seeing a 2nd rdg by itself
-     (r'^1st rdg - to 2nd rdg', "bill:reading:2"),
-     # second to third will count as a third read if there is no
-     # explicit third reading action
-     (r'2nd rdg - to 3rd rdg', "bill:reading:3"),
-     (r'^3rd rdg$', "bill:reading:3"),
-     (r'.*Third Time.*PASSED.*', ["bill:reading:3", "bill:passed"]),
-     # bill:reading:3, bill:passed
-     (r'^3rd rdg as amen - (ADOPTED|PASSED)', ["bill:reading:3", "bill:passed"]),
-     (r'^3rd rdg - (ADOPTED|PASSED)', ["bill:reading:3", "bill:passed"]),
-     (r'^Read Third Time in Full .* (ADOPTED|PASSED).*', [
-         "bill:reading:3", "bill:passed"]),
-     (r'^.*read three times - (ADOPTED|PASSED).*', [
-         "bill:reading:3", "bill:passed"]),
-     (r'^.*Read in full .* (ADOPTED|PASSED).*', [
-         "bill:reading:3", "bill:passed"]),
-     # bill:reading:3, bill:failed
-     (r'^3rd rdg as amen - (FAILED)', ["bill:reading:3", "bill:failed"]),
-     (r'^3rd rdg - (FAILED)', ["bill:reading:3", "bill:failed"]),
-     # rules suspended
-     (r'^Rls susp - (ADOPTED|PASSED|FAILED)', lambda mch, ch: {
-                                                       'ADOPTED': "bill:passed",
-                                                        'PASSED': "bill:passed",
-                                                        'FAILED': "bill:failed"
-                                                   }[mch.groups()[0]]),
-     (r'^to governor', "governor:received"),
-     (r'^Governor signed', "governor:signed"),
+    # bill:reading:1
+    (r'(\w+) intro - (\d)\w+ rdg - to (\w+/?\s?\w+\s?\w+)',
+     lambda mch, ch: ["bill:introduced", "bill:reading:1", "committee:referred"] \
```

Comment:
Some of these aren't valid actions in pupa. Can you run `scripts/convert-actions.sh` on this file to update the actions?